server/embedded/src/org/labkey/embedded/LabKeyServer.java

```diff
@@ -405,7 +404,6 @@ public static class ContextProperties
 
         private String webAppLocation;
         private String workDirLocation;
-        @NotNull (message = "Must provide encryptionKey")
```

Comment:
Fine if we want to eliminate the dependency on `NotNull` but presumably we should still enforce that the value is not null. Add a check in `getEncryptionKey()` and throw an `IllegalStateException` or similar?

Reply:
Done